ospf6d/ospf6_interface.h

```diff
@@ -182,7 +182,7 @@ extern const char *const ospf6_interface_state_str[];
 #define OSPF6_INTERFACE_TRANSDELAY     1
 #define OSPF6_INTERFACE_INSTANCE_ID    0
 #define OSPF6_INTERFACE_BANDWIDTH      10000   /* Mbps */
-#define OSPF6_REFERENCE_BANDWIDTH      100000  /* Mbps */
+#define OSPF6_REFERENCE_BANDWIDTH 100000       /* Kbps */
```

Comment:
why the change in alignment with the rest of the code?

Reply:
ok